Pinta.Core/Classes/Layer.cs

```diff
@@ -77,15 +80,20 @@ public Layer (ImageSurface surface, bool hidden, double opacity, string name)
 			set { if (hidden != value) SetValue (HiddenProperty, ref hidden, value); }
 		}
 		
+		public bool Locked {
```

Comment:
I wrote this essentially so that it would pass through any locking information if it was set. Probably best to comment out the checkbox in the dialog. 

Locking can be confusing for users if it is is not made very very clear why they can't do what they are trying to do (even having a menu greyed out can be confusing, and not obvious how to get to the feature). 

Lots of these little things got written while I was taking a break from trying to figure out the things I couldn't get to work.